widget/list.go

```diff
@@ -213,7 +213,7 @@ func (l *listRenderer) Layout(size fyne.Size) {
 }
 
 func (l *listRenderer) MinSize() fyne.Size {
-	return l.scroller.MinSize()
+	return l.scroller.MinSize().Max(l.list.itemMin)
```

Comment:
Does itemMin include the padding amount for the indicator?

Reply:
Yes

Reply:
Yup